api/src/State/ChecklistCreateProcessor.php

```diff
@@ -20,6 +25,17 @@ public function __construct(ProcessorInterface $decorated) {
      * @param Checklist $data
      */
     public function onBefore($data, Operation $operation, array $uriVariables = [], array $context = []): Checklist {
+        if (!$data->isPrototype && null == $data->camp) {
+            throw new ValidationException(new ConstraintViolationList([
+                new ConstraintViolation('This value should not be null.', '', [], null, 'camp', null, null, null, new NotNull()),
```

Comment:
Can we discuss if this should not be a validator directly an the entity?
Either via a custom validator or via @Assert\Expression?
Alternative idea: not storing `isPrototype` in database but use it as calculated value only.

Reply:
I try to use `Assert\Expression`